### PR TITLE
test(e2e): check-xpub-ADA

### PR DESCRIPTION
**Pull request by bosomt**
check of presence of Cardano ADA xpub in account detail.

### CODE REVIEW

**Comment by szymonlesisz** [comment]
please change the commit message (see QA slack thread)

File: packages/integration-tests/projects/suite-web/tests/wallet/check-xpub-ada.test.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    /* Test case
+    1. Pass onboarding
+    2. Disable BTC coin to speedup discovery process
+    3. Enable Ada coin
+    4. Navigate to Accounts/ADA/Account/
+    5. Click on @wallets/details/show-xpub-button
+    6. Check that @xpub-modal/xpub-field is present
+    */
+    it.only('Check Cardano XPUB', () => {
```
Comment (mroz22) [inline packages/integration-tests/projects/suite-web/tests/wallet/check-xpub-ada.test.ts]: you most likely don't want to leave `only` here.

Comment (bosomt) [reply packages/integration-tests/projects/suite-web/tests/wallet/check-xpub-ada.test.ts]: @mroz22 done. sorry i will check it next time ;)

**Comment by ondracja** [comment]
@bosomt please rebase the branch and update the commit message.

**Review by ondracja — APPROVED** [review]

**Pull request merged**